README.md

```diff
@@ -1,4 +1,4 @@
-# flexy-osisoft
```

Comment:
Whatever editor you used added the BOM.  This should be removed.

Reply:
It should be removed now

Reply:
You will want to remove the change from the first commit.  There are a number of ways to do this in git, the easiest in this case is to probable rebase and apply your 2nd commit as a "fixup" to the first commit.

As a general rule anything introduced in a pull request should not be removed in the same pull request.  The idea being that everything in the pull request is 100% intentional.

Reply:
I got it, thanks.
It's definitely good to pick that stuff up now